python/test/kv/kvTest.py

```diff
@@ -24,8 +26,24 @@ def testSimple(mode):
         print("Got: ", aFetched)
         return False
 
-    if mode == 'Anna':
-        return True
+    #if mode == 'Anna':
+    #    return True
+    if mode == 'sharemem':
```

Comment:
What is this special extra part for? Does this test something sharemem specific?

Reply:
Yes. It's testing whether the index increases correctly so that we can get the value from the middle of our shared memory.